tests/.run-doctests.sh

```diff
@@ -5,5 +5,5 @@ set -e -u -o pipefail
 # > pytest --doctest-modules speechbrain/
 # However, we take this more complex approach to avoid testing files not
 # tracked by git. We filter out tests that require optional dependencies.
-avoid="transducer_loss.py\|fairseq_wav2vec.py\|huggingface_wav2vec.py\|bleu.py\|ctc_segmentation.py\|check_url.py\|huggingface_whisper.py\|language_model.py\|vocos.py\|discrete_ssl.py\|lobes/flair|lobes/spacy"
+avoid="transducer_loss.py\|fairseq_wav2vec.py\|huggingface_wav2vec.py\|bleu.py\|ctc_segmentation.py\|check_url.py\|huggingface_whisper.py\|language_model.py\|vocos.py\|discrete_ssl.py\|lobes/models/flair\|lobes/models/spacy\|llama2.py"
```

Comment:
I've tried modifying this line recently, and it was not working (See the PR on reforging Llama).

Reply:
There's two places that tests need to be excluded, here and `conftest.py`